vignettes/EnrichmentBrowser.Rmd

```diff
@@ -0,0 +1,989 @@
+renders five gene sets significant and the remaining insignificant.
+
+```{r dummySBEA}
+ dummySBEA <- function(se, gs) {
+   sig.ps <- sample(seq(0, 0.05, length = 1000), 5) 
+   insig.ps <- sample(seq(0.1, 1, length = 1000), length(gs) - 5) 
+   ps <- sample(c(sig.ps, insig.ps), length(gs)) 
+   names(ps) <- names(gs) 
+   return(ps)
+ }
+```
+
+We can plug this method into `sbea` as before.
+
+```{r sbea2}
+sbea.res2 <- sbea(method = dummySBEA, se = allSE, gs = hsa.gs) 
+gsRanking(sbea.res2)
+```
+
+As described in the previous section, it is also possible to analogously
+plug in user-defined network-based enrichment methods into `nbea.`
+
+# Combining results
+
+Different enrichment analysis methods usually result in different gene
+set rankings for the same dataset. To compare results and detect gene
+sets that are supported by different methods, the `r Biocpkg("EnrichmentBrowser")` package allows to
+combine results from the different set-based and network-based
+enrichment analysis methods. The combination of results yields a new
+ranking of the gene sets under investigation by specified ranking
+criteria, e.g. the average rank across methods. We consider the ORA
+result and the GGEA result from the previous sections and use the
+function `combResults`.
+
+```{r combine}
+res.list <- list(sbea.res, nbea.res) 
+comb.res <- combResults(res.list)
+```
+
+The combined result can be detailedly inspected as before and
+interactively ranked as depicted in Figure \@ref(fig:browsecomb).
+
+```{r browse-comb, eval=FALSE}
+ eaBrowse(comb.res, graph.view=hsa.grn,
+nr.show=5)
+```
+
+
+![(#fig:browsecomb) Combined result view
+By clicking on one of the columns (ORA.RANK, ..., GGEA.PVAL) the result can be interactively ranked according to the selected criterion.](comb_ebrowse.png){width="20cm" height="6.8cm"}
+
+
+# Putting it all together
+
+There are cases where it is necessary to perform certain steps of the
+demonstrated enrichment analysis pipeline individually. However, it is
+often more convenient to run the complete standardized pipeline. This
+can be done using the all-in-one wrapper function `ebrowser`. For example, the
+result page displayed in Figure \@ref(fig:browsecomb) can also be produced from scratch via
+
+```{r all-in-one, eval=FALSE}
+ebrowser( meth=c("ora", "ggea"), exprs=exprs.file, cdat=cdat.file, rdat=rdat.file, org="hsa", gs=hsa.gs, grn=hsa.grn, comb=TRUE, nr.show=5)
+```
+
+# Advanced: configuration parameters {#sec:config}
+
+Similar to 's options settings, the `r Biocpkg("EnrichmentBrowser")` uses certain package-wide configuration parameters, which affect the way in which analysis is carried out and how results are displayed. The settings of these parameters can be examined and, to some extent, also changed using the function `configEBrowser`. For instance, the default directory where the `r Biocpkg("EnrichmentBrowser")` writes results to can be updated via
```

Comment:
Missing `R`. It should read: `Similar to R's options`

Reply:
k